index.html

```diff
@@ -289,6 +289,22 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact-section" id="contact">
+        <div class="contact-form-container">
+          <p class="contact-form-heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+          <form action="https://formspree.io/f/xrgjbnyl" method="post" class="contact-form">
+            <div class="contact-form-name">
+              <input type="text" name="firstName" placeholder="First Name">
+              <input type="text" name="lastName" placeholder="Last Name">
```

Comment:
- [ ] Kindly add `required` and `maxlength="30"` attributes to the `First Name` & `Last Name` input field.